src/Core/Abstractions/ISyncService.cs

```diff
@@ -7,7 +7,7 @@ namespace Bit.Core.Abstractions
     public interface ISyncService
     {
         bool SyncInProgress { get; set; }
-
+        Task<bool> IsSyncOnRefreshEnabledAsync();
```

Comment:
We have some longer-term goals to keep the `Core` project very non-platform specific and use it for more clients. Since a pull to refresh feature is very mobile-specific, I'd like it if this were not in a Core service. Can we just call the storage service each time this is needed instead of having a service function for it? Looks like a 1 liner anyways.

Reply:
I did not know about the non-platform specific in core, i thought it was better to avoid duplicate code. I made the changes.